app.rb

```diff
@@ -0,0 +1,77 @@
+require_relative 'library'
+
+class ConsoleApp
+  def initialize
+    @library = Library.new
+  end
+
+  def all_books
+    puts 'List of Books:'
+    @library.books.each { |book| puts "#{book.title} by #{book.author}" }
+  end
+
+  def all_people
+    puts 'List of People:'
+    @library.people.select { |key| puts key[:name] }
+  end
+
+  def add_person
+    print 'Enter person\'s name: '
+    name = gets.chomp
+    person = @library.create_person({ name: name })
+    person.name.each do |key, value|
+      puts "Created person #{key} as #{value}"
+    end
+  end
```

Comment:
- [ ] Please make sure that your application functions similarly to the one demonstrated in the provided [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), as per the reviewers' request. When adding a new person, it is important to display the prompt shown in the video, asking whether the user intends to create a student or a teacher. As you work on this kindly ensure that all features of your app are working correctly, just like in the given [video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) :blush: